text-ui-test/EXPECTED.TXT

```diff
@@ -15,6 +15,9 @@ Okay! I have added this module to your module list:
 Name: Software Engineering & Object-Oriented Programming | Sem: 19-20 Semester 2
 
 
+Okay, i've marked the module as done!
+
+
```

Comment:
I think the automatically checks were not successful because of these three lines, it seems that there is no "done" command in input.txt 😂